From 786609c8aef0867ce2f8f0dcac2b5978ef063426 Mon Sep 17 00:00:00 2001
From: Charlie Root <root@jdfbsd12.multipass>
Date: Sun, 13 Sep 2020 19:04:47 -0400
Subject: [PATCH 2/5] added belkin b2b128 support

---
 sys/dev/usb/usbdevs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sys/dev/usb/usbdevs b/sys/dev/usb/usbdevs
index 2b25aa0ee..a79272839 100644
--- a/sys/dev/usb/usbdevs
+++ b/sys/dev/usb/usbdevs
@@ -1366,6 +1366,7 @@ product BELKIN F5U109		0x0109	F5U109 Serial
 product BELKIN USB2SCSI		0x0115	USB to SCSI
 product BELKIN F8T012		0x0121	F8T012xx1 Bluetooth USB Adapter
 product BELKIN USB2LAN		0x0121	USB to LAN
+product BELKIN B2B128		0x0128	B2B128 USB3 to LAN
 product BELKIN F5U208		0x0208	F5U208 VideoBus II
 product BELKIN F5U237		0x0237	F5U237 USB 2.0 7-Port Hub
 product BELKIN F5U257		0x0257	F5U257 Serial
-- 
2.27.0

